package/zshelf/package

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(zshelf)
+pkgdesc="Z-Library browser and downloader"
+url=https://github.com/khanhas/zshelf
+pkgver=0.1.1
+section=utils
+timestamp=2021-01-31T05:25Z
+maintainer="khanhas <xuankhanh963@gmail.com>"
+license=GPL-3.0
+depends=(node)
+
+source=(
+    https://github.com/khanhas/zshelf/archive/v0.1.1.zip
+    zshelf.draft
+    zshelf.png
+)
+sha256sums=(
+    c6bc1980c90e8ea05149f3bd609667edba4e8465f9db0f6882407ca4dc70e120
+    SKIP
+    SKIP
+)
+
+image=qt:v1.1
+build() {
+    qmake zshelf.pro
+    make
+
+    # Get needed build packages
+    export DEBIAN_FRONTEND=noninteractive
+    apt-get update -y
+    apt-get install -y \
+        --no-install-recommends \
+        -o Dpkg::Options::="--force-confdef" \
+        nodejs npm
+
+    cd backend
+    npm install
+}
+
+package() {
+    install -d "$pkgdir"/opt/share/zshelf
+
+    cp -r "$srcdir"/backend "$pkgdir"/opt/share/zshelf/backend
+    cp "$srcdir"/config.json "$pkgdir"/opt/share/zshelf
+    install -D -m 755 -t "$pkgdir"/opt/share/zshelf "$srcdir"/zshelf
+
+    ln -s /opt/share/zshelf/zshelf "$pkgdir"/opt/bin
```

Comment:
```suggestion
    install -d "$pkgdir"/opt/bin
    ln -s /opt/share/zshelf/zshelf "$pkgdir"/opt/bin/zshelf
```

Before `/opt/bin` was the symlink to the binary which at best is a broken package and at worst one that can break others (luckily opkg doesn't allow it to overwrite/delete the `/opt/bin` directory). The suggestion above fixes that and the package should run just fine then.